### PR TITLE
Add No robots

### PR DESCRIPTION
**Pull request by vumichien**
Hi @shayne-longpre

This PR adds the No Robots Dataset (with the downloader, preparer, and mapper). All tests passed successfully.

Tell me if you need to change anything.

### CODE REVIEW

File: data_summaries/No Robots.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            "Commonsense Classification",
+            "Closed Question Answering",
+            "Information Extraction"
+        ],
+        "Text Sources": [
+            "human"
+        ],
+        "Model Generated": [],
+        "Format": [
+            "Chain-of-Thought"
```
Comment (shayne-longpre) [inline data_summaries/No Robots.json:34]: zero-shot?

File: data_summaries/No Robots.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        ],
+        "Human Annotation": "Yes",
+        "Derived from Datasets": [""],
```
Comment (shayne-longpre) [inline data_summaries/No Robots.json:37]: Just empty list I believe

File: data_summaries/PII-masking-200k.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
+{
```
Comment (shayne-longpre) [inline data_summaries/PII-masking-200k.json:1]: I think the 2 PRs got entangled as this is already in another one?

**Review by shayne-longpre — APPROVED** [review]
Approved once the fixes are made, and it passes all tests without warnings.

File: data_summaries/No Robots.json (hunk 1 above)
Comment (vumichien) [reply data_summaries/No Robots.json]: It is revised as a comment.

File: data_summaries/No Robots.json (hunk 2 above)
Comment (vumichien) [reply data_summaries/No Robots.json]: It is revised as a comment.

File: data_summaries/PII-masking-200k.json (hunk 3 above)
Comment (vumichien) [reply data_summaries/PII-masking-200k.json]: disentangle this PR from privacy-200

**Comment by shayne-longpre** [comment]
@vumichien looks like the No Robots data summary and parts were accidentally removed?

**Comment by vumichien** [comment]
@shayne-longpre, I have added No Robots data summary again, sorry for my mistake. The conflicts were resolved

**Pull request merged**